.github/workflows/sonarcloud.yml

```
@@ -31,7 +31,7 @@ jobs:
     - name: Install 'cargo-sonar'
       run: cargo install cargo-sonar
```

Maybe, if you want to avoid breaking change later, you can do the following (depending if you prefer to live on the edge and be ok with changes like this one, or prefer to be more stable !).

```suggestion
      run: cargo install cargo-sonar --version 0.9.0
```

---

Thanks for the tip :+1: